### ISSUE TITLE
Is outdated example?

### ISSUE THREAD

**Issue by KES777** [Corion/WWW-Mechanize-Chrome#23]
https://metacpan.org/pod/WWW::Mechanize::Chrome::Examples#Example:-html-to-pdf.pl

This does not save generated content into file.

**Comment by KES777** [comment]
I did this manually:
```perl
...
use Mojo::File;
my $file =  Mojo::File->new( 'test.pdf' );
$file->spurt( $page_pdf );

```

**Comment by Corion** [comment]
Sorry - the documentation was taken over from WWW::Mechanize::Firefox / WWW::Mechanize::PhantomJs , but the code was never implemented properly. This is now fixed by b7f01a3e4113ce3e84e5b376de4106fb9442bed4 .

**Comment by KES777** [comment]
thank you
